carbonmark/lib/projectGetter.ts

```diff
@@ -13,6 +14,21 @@ export const getCategoryFromProject = (
   return asCategoryName(name); // fallback for Staging Testnet Data
 };
 
+export const getCategoriesFromProject = (
+  project: DetailedProject
+): CategoryName[] => {
+  const methodologies = compact(project.methodologies);
+  return !isEmpty(methodologies)
+    ? Array.from(
+        new Set(
+          methodologies.map(
+            (methodology) => methodology.category as CategoryName
+          )
+        )
+      )
+    : ["Other"];
+};
```

Comment:
Just a suggestion but Lodash [uniq](https://lodash.com/docs/4.17.15#uniq) or [uniqBy](https://lodash.com/docs/4.17.15#unionBy) could make this a lot easier.

```ts
  const fn = pipe(map(get("category")), compact, uniq);
  const methodologies = fn(project.methodologies);
  return !isEmpty(methodologies) ? methodologies : ["Other"];
```

Reply:
Thanks for the tip! I need to use lodash more.

Reply:
bc72481aa